ios/Podfile.lock

```diff
@@ -696,4 +702,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 41b806c7f131f87b716be1f1f9377532d6c9e43a
 
-COCOAPODS: 1.10.0
+COCOAPODS: 1.10.1
```

Comment:
I have had to run `pod install` every time I want to run iOS for months, and the diff always looks like this (at least with the updated COCOAPODS version). I've never committed the updated `Podfile.lock`, because I'm not sure if I had a different cocoapods version than I was supposed to, or what the story was here.

Reply:
It should always match the [`cocoapods version in our Gemfile.lock`](https://github.com/Expensify/Expensify.cash/blob/master/Gemfile.lock#L35)